### PR TITLE
add custom expiration property & update gemspec for Rails 5 compatibility

### PR DESCRIPTION
**Pull request by snicky**

### CODE REVIEW

**Comment by snicky** [comment]
@jcham @jimytc could you help me to review this PR?

File: lib/ardis/base_series.rb [hunk 1]
```diff
@@ -42,6 +42,7 @@ def initialize( name:             nil,
                     initializer:      nil,
                     inverse_of:       nil,
                     decorator:        nil,
+                    expiration:       nil,
```
Comment (jimytc) [inline lib/ardis/base_series.rb:45]: @snicky 
I'm not sure why putting `expiration` for a series.
I guess ardis is to make it works like ActiveRecord so I don't think it's good to expire data implicitly.
Maybe I mis-understand it.

@jcham 
Do you have any idea?

**Comment by snicky** [comment]
@jimytc I find it useful to define a series sorted by a score that might change over time, e.g.:

```
    series_sorted_set name: 'similar', relation: Picture, expiration: 1.day,
      attr_score: -> (picture) { -picture.voting },
      initializer: -> (picture) {
        picture.find_related_tags
      }
```

In this case my `picture.similar` works as a cached collection that can be rebuilt only once a day. I have actually kept this as a list of ids in memcached before, but I realized ardis is better for this task as there's no need to write any additional code for loading and paginating the data from the database.

**Comment by jimytc** [comment]
Hmm. Okay

**Review by jimytc — APPROVED** [review]

**Pull request merged**